synapse/storage/databases/main/events_worker.py

```diff
@@ -1462,68 +1468,68 @@ async def have_seen_events(
             event_ids: events we are looking for
 
         Returns:
-            The set of events we have already seen.
+            The remaining set of events we haven't seen.
```

Comment:
Changed from returning the events we saw to the events we didn't see.

In big rooms like `#matrixhq`, we've seen most of the events already. Only a little piece of the state is new.